.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
```

Comment:
This has to be reverted before this change is submitted and after the test-changes are submitted.

Reply:
Done.